distribute.sh

```diff
@@ -93,6 +93,11 @@ case $OSTYPE in
 		;;
 esac
 
+if [ -f /proc/cpuinfo ]; then
```

Comment:
Maybe this should check if `MAKE_JOBS` is already set, so you can tell it how many jobs to use? Like `if [ -z "$MAKE_JOBS" -a -f /proc/cpuinfo ]; then` ...